packages/frontend/src/utils/ledgerManager.js

```diff
@@ -0,0 +1,72 @@
+import { listen } from '@ledgerhq/logs';
+import bs58 from 'bs58';
+import { createClient, getSupportedTransport, setDebugLogging } from 'near-ledger-js';
+
+setDebugLogging(true);
+listen(console.log);
+
+window.Buffer = Buffer; // Exists due to `bs58` import
+
+class LedgerManager {
+    constructor() {
+        this.bs58 = bs58;
```

Comment:
This doesn't appear to be referenced anywhere. Do we need it?

Reply:
Right, we don't use it. Done.